src/integrationTests/options.test.ts

```diff
@@ -209,11 +219,332 @@ describe('useCookie option', () => {
   });
 });
 
+describe('multiple init calls', () => {
+  const identity = makeIdentity();
+  const baseUrl = 'http://test-host';
+  const cookiePath = '/test/';
+  const cookieDomain = 'uidapi.com';
+
+  describe('when nothing has changed', () => {
+    beforeEach(() => {
+      uid2.init({
+        identity: identity,
+        baseUrl: baseUrl,
+        cookiePath: cookiePath,
+      });
+      uid2.init({
+        identity: identity,
+        baseUrl: baseUrl,
+        cookiePath: cookiePath,
+      });
+      uid2.init({
+        identity: identity,
+        baseUrl: baseUrl,
+        cookiePath: cookiePath,
+      });
+    });
+    test('should return next two init calls without changing anything', () => {
+      expect(getUid2LocalStorage().advertising_token).toBe(identity.advertising_token);
+      let storageConfig = getConfigStorage();
+      expect(storageConfig).toBeInstanceOf(Object);
+      expect(storageConfig).toHaveProperty('cookiePath', cookiePath);
+    });
+  });
+
+  describe('new base URL is given', () => {
+    const oldBaseUrl = baseUrl;
+    const newBaseUrl = 'http://example';
+
+    beforeEach(() => {
+      uid2.init({
+        callback: callback,
+        identity: identity,
+        baseUrl: oldBaseUrl,
+        useCookie: true,
+      });
+    });
+    test('should use new base url', () => {
+      uid2.init({
+        baseUrl: newBaseUrl,
+      });
+      const configCookie = getConfigCookie();
+      expect(configCookie).toHaveProperty('baseUrl', newBaseUrl);
+    });
+
+    test('should use old base url', () => {
```

Comment:
This test should probably belong somewhere else. Looking up to the describe it's in, it's 'new base URL is given' - it doesn't make sense that it 'should use old base url'. Perhaps you could move 'new base URL is given' into the previous test, and this one could be 'no base URL is given, should use the original base URL'.